controllers/fetchers/grafana_com_fetcher.go

```diff
@@ -42,7 +42,9 @@ func getLatestGrafanaComRevision(dashboard *v1beta1.GrafanaDashboard) (int, erro
 		return -1, err
 	}
 
-	client := client2.NewInstrumentedRoundTripper(fmt.Sprintf("%v/%v", dashboard.Namespace, dashboard.Name), metrics.GrafanaComApiRevisionRequests, true)
+	// insecure to true because we don't know if the target URL is recognized by the default certificate
+	tlsConfig := client2.BuildInsecureTLSConfiguration()
```

Comment:
This should use a secure configuration as it always fetches from `https://grafana.com` (which should be verified)

Reply:
I have try to make a default version of this one. I'm currently testing it.

Reply:
The test is working with a default configuration. I push this chnage.